src/style.css

```
@@ -0,0 +1,78 @@
+@font-face {
+  font-family: "Avenir";
+  src: local("Avenir"), url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+[data-qa] {
+  width: 100%;
+  margin-bottom: 20px;
+  margin-top: 20px;
+}
+
+.input {
+  display: block;
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  background: #fff;
+  color: #3d4e61;
+  font-family: "Avenir", sans-serif;
```

set font family to the `html` tag selector and let inputs inherit it 
```suggestion
  font-family: inherit
```